### PR TITLE
adds contentMinHeight prop

### PR DESCRIPTION
**Pull request by tsargent**

### CODE REVIEW

**Comment by jasonfellows** [comment]
:+1:

Listing all props in the propTypes would be good too

**Comment by loudhearted** [comment]
:+1:

File: src/js/components/modal/modal.js [hunk 1]
```diff
@@ -78,7 +79,7 @@ export default React.createClass({
   render() {
     return <div ref="modal" style={{zIndex: this.props.zIndex}} className={this.modalBackgroundClasses()} onClick={this.clickBackground}>
         <div className="container fill">
-          <div onClick={this.clickContent} className={this.modalContentClasses()} style={{height: this.props.contentHeight}}>
+          <div onClick={this.clickContent} className={this.modalContentClasses()} style={{height: this.props.contentHeight, minHeight: this.props.contentMinHeight}}>
```
Comment (HipsterBrown) [inline src/js/components/modal/modal.js]: If this is optional, what happens when it's not passed in? Is there a default value?

Comment (HipsterBrown) [inline src/js/components/modal/modal.js]: @tsargent ^

**Pull request merged**